fulp_modules/ipc/code/organs/lungs.dm

```diff
@@ -0,0 +1,30 @@
+/obj/item/organ/lungs/robot_ipc
+	name = "heat sink"
+	desc = "A device that transfers generated heat to a fluid medium to cool it down. Required to keep your synthetics cool-headed. It's shape resembles lungs." //Purposefully left the 'fluid medium' ambigious for interpretation of the character, whether it be air or fluid cooling
+	icon = 'fulp_modules/ipc/icons/ipc_organs.dmi'
+	icon_state = "lungs-ipc"
+	safe_nitro_min = 0
+	safe_nitro_max = 0
+	safe_co2_min = 0
+	safe_co2_max = 0
+	safe_toxins_min = 0
+	safe_toxins_max = 0
+	safe_oxygen_min = 0	//What are you doing man, dont breathe with those!
+	safe_oxygen_max = 0
+	cold_level_1_damage = 0
+	cold_level_2_damage = 0
+	cold_level_3_damage = 0
+	status = ORGAN_ROBOTIC
+	organ_flags = ORGAN_SYNTHETIC
+	slot = ORGAN_SLOT_LUNGS
+
+/obj/item/organ/lungs/robot_ipc/emp_act(severity)
```

Comment:
do TRAIT_NOBREATH species even need lungs? if they don't, then wouldn't it be beneficial for an IPC to get their lungs removed at roundstart so that they don't overheat when EMP'd?